### PR TITLE
meta(typo): Fix multiple typos

### PR DESCRIPTION
**Pull request by billyvg**
This is a continuation of @minho42 work on https://github.com/getsentry/sentry/pull/14615, rebased on top of our latest master and resolved the merge conflicts. 

Closes #14615

### CODE REVIEW

**Comment by billyvg** [comment]
IDK if there's an easier way for me to help resolve the conflicts in https://github.com/getsentry/sentry/pull/14615

File: src/sentry/db/models/query.py [hunk 1]
```diff
@@ -94,8 +94,8 @@ def create_or_update(model, using=None, **kwargs):
 
 
 def in_iexact(column, values):
-    """Operator to test if any of the given values are (case-insentive) matches
-       to values in the given column."""
+    """Operator to test if any of the given values are (case-insensitively)
```
Comment (dashed) [inline src/sentry/db/models/query.py]:
```suggestion
    """Operator to test if any of the given values are (case-insensitive)
```

File: src/sentry/db/models/query.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def in_iexact(column, values):
 
 
 def in_icontains(column, values):
-    """Operator to test if any of the given values are (case-insentively)
+    """Operator to test if any of the given values are (case-insensitively)
```
Comment (dashed) [inline src/sentry/db/models/query.py]:
```suggestion
    """Operator to test if any of the given values are (case-insensitive)
```

**Review by dashed — CHANGES_REQUESTED** [review]

File: src/sentry/db/models/query.py (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/db/models/query.py]: I think this is fine as it is.

Comment (EvanPurkhiser) [reply src/sentry/db/models/query.py]: 🤷‍♂

Comment (dashed) [reply src/sentry/db/models/query.py]: 🤔

**Review by dashed — APPROVED** [review]
Looks good to me 👍 

Took me a while to look thru them all 😄

**Review by tkaemming — APPROVED** [review]
🙃

**Comment by joshuarli** [comment]
Thanks so much @minho42! Also billy.

**Comment by minho42** [comment]
@billyvg Thank you!

**Pull request merged**